acquisitions/src/main/resources/thunderjet/mod-orders/features/open-order-with-resolution-statuses.feature

```diff
@@ -0,0 +1,61 @@
+# For MODORDERS-1170
+Feature: Close order if order line has resolution statuses that should make it as closed
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': '*/*' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+    * def orderId = callonce uuid3
+    * def poLineId = callonce uuid4
+
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
```

Comment:
We have these reusable features defined in karate-config.js already, no need to define again
https://github.com/folio-org/folio-integration-tests/blob/e0ef0143cd7c72ab2b61fcbb933615169a7470b7/acquisitions/src/main/resources/karate-config.js#L66